### ISSUE TITLE
Use hashes with git mode on Windows

### ISSUE THREAD

**Issue by nake89** [tiged/tiged#43]
If you look here code will run differently on windows than on other OS: https://github.com/tiged/tiged/blob/main/src/index.js#L334

The way to be get tiged to work with commit hashes on git mode is to do command line calls which obviously don't work on Windows. If somebody wants to make this work (or if I ever want to boot up a Windows OS, unlikely), feel free to submit PR.

**Comment by nake89** [comment]
@aryaemami59 is this fixed? I think you added Windows to the github actions. And the tests pass. Does that mean this can be closed?

**Comment by aryaemami59** [comment]
@nake89 Yes, this was fixed by the changes made in #89 specifically [this commit](https://github.com/tiged/tiged/pull/89/commits/91ad763f34bff3e68e834bd4a770bb1e8af049f1).

**Comment by nake89** [comment]
lol. totally forgot about that. Thanks :D
